dask/tests/test_distributed.py

```diff
@@ -289,7 +289,6 @@ def test_to_hdf_distributed(c):
         ),
     ],
 )
-@pytest.mark.xfail_with_pyarrow_strings
```

Comment:
This still seems to be problematic for me; I'm not sure why it could be removed.